src/modules/job-manager/event.c

```diff
@@ -434,6 +438,27 @@ static int event_release_context_decode (json_t *context,
     return 0;
 }
 
+static int event_handle_dependency (struct job *job,
+                                    const char *cmd,
+                                    json_t *context)
+{
+    const char *name;
+
+    if (json_unpack (context, "{s:s}", "name", &name) < 0) {
```

Comment:
in RFC21 the "name" is actually "description"?

Reply:
Great catch, thanks! I'll update `name` to `description`